boefjes/boefjes/plugins/kat_dns/main.py

```diff
@@ -17,44 +18,50 @@ class ZoneNotFoundException(Exception):
     pass
 
 
+def get_record_types() -> List[str]:
+    dns_record_types = ["A", "AAAA", "CAA", "CERT", "RP", "SRV", "TXT", "MX", "NS", "CNAME", "DNAME"]
```

Comment:
Can be a constant `DNS_RECORD_TYPES` (and as a `set` type) outside of the function